app/Helpers/database/player-achievement.php

```diff
@@ -183,8 +183,13 @@ function unlockAchievement(string $username, int $achievementId, bool $isHardcor
 
 function getAchievementUnlockCount(int $achID): int
 {
-    $query = "SELECT COUNT(*) AS NumEarned FROM Awarded
-              WHERE AchievementID=$achID AND HardcoreMode=0";
+    if (env('FEATURE_AGGREGATE_QUERIES')) {
+        $query = "SELECT COUNT(*) AS NumEarned FROM player_achievements
+                  WHERE achievement_id=$achID";
```

Comment:
This function is only called when promoting/demoting achievements. I couldn't get an accurate reading on the timing, but running the query from within mysql:

**achievement/3159**
With feature: 4ms
Without feature: 6ms